.github/workflows/publishDocumentation.yml

```diff
@@ -0,0 +1,30 @@
+name: PublishDocumentation
+on:
+  push:
```

Comment:
We only want to publish when we merge to `main` (default branch) — i.e. when new commits show up on `main`.

```suggestion
on:
  push:
    branches: [ $default-branch ]
```

Oddly, I haven't been able to get the `branches: [ $default-branch ]` to work. Might be that `$default-branch` has issues? Maybe try `branches: [ main ]` if the `$default-branch` doesn't work?

Reply:
Yeah, was going to ask about that. Will fix.